functions/snippets/src/main/java/Tips.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START functions_tips_scopes]
+// [START run_tips_global_scope]
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class Tips {
+
+  // Create a client with some reasonable defaults. This client can be reused for multiple requests.
+  private static HttpClient client =
+      HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(10)).build();
+
+  // [END functions_tips_scopes]
```

Comment:
Excluding these functions breaks one of the goals of the new sample format, in that the sample isn't copy-paste-runnable. Can you leave them in (but maybe move them towards the bottom of the class since they aren't important?)

Reply:
Sure.
I was just following the other samples that exclude them.
Can include them at the bottom.